pkg/code/data/nonce/postgres/model.go

```diff
@@ -17,13 +18,15 @@ const (
 )
 
 type nonceModel struct {
-	Id        sql.NullInt64 `db:"id"`
-	Address   string        `db:"address"`
-	Authority string        `db:"authority"`
-	Blockhash string        `db:"blockhash"`
-	Purpose   uint          `db:"purpose"`
-	State     uint          `db:"state"`
-	Signature string        `db:"signature"`
+	Id               sql.NullInt64 `db:"id"`
+	Address          string        `db:"address"`
+	Authority        string        `db:"authority"`
+	Blockhash        string        `db:"blockhash"`
+	Purpose          uint          `db:"purpose"`
+	State            uint          `db:"state"`
+	Signature        string        `db:"signature"`
+	ClaimNodeId      string        `db:"claim_node_id"`
+	ClaimExpiresAtMs int64         `db:"claim_expires_at"`
```

Comment:
From the docs[1], as long as we allow these two fields to be `NULL` (and have it as the default), then there won't need to be any rewrite of the underlying tables. This means the `ALTER TABLE` command _will_ place an exclusive access on the table, but the duration of the command _should_ be quick.